src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
+        Excavator excavator = new Excavator();
```

Comment:
```suggestion
        Machine excavator = new Excavator();
```
краще оголошувати змінні з посиланням інтерфейсу